### PR TITLE
feat: build plugin pages and update electron ver.

### PR DESCRIPTION
**Pull request by agatha197**
Plugin pages are not accessible from the electron app. This PR builds both the components and plugins folder.

## How to test
1. add any plugin page
2. set plugins - page of the `config.toml` file
     e.g) page = "link-test-plugin,admin-plugin"
3. make clean
4. make dep
5. npm run electron:d

## Checklist for reviewer
- [ ] Can you access plugin pages from the electron app?

Just in case, this is my plugin test page.
set `page = "link-test-plugin"` of the `config.toml` file
```typescript
/**
 @license
 Copyright (c) 2015-2024 Lablup Inc. All rights reserved.
 */
import { BackendAIPage } from '../components/backend-ai-page';
import { html } from 'lit';
import { customElement, property } from 'lit/decorators.js';

/**
 Test plugin for Backend.AI Console

 */
@customElement('link-test-plugin')
export default class LinkTestPlugin extends BackendAIPage {
  @property({ type: String }) menuitem = 'LinkTestPage'; // Menu name on sidebar.
  @property({ type: String }) is = 'link-test-plugin'; // Should be exist.
  @property({ type: String }) permission = 'user'; // Can be 'user', 'admin' or 'superadmin'.
  @property({ type: String }) url = 'https://www.google.com'; // URL to redirect.

  constructor() {
    super();
  }

  static get styles() {
    return [];
  }

  updated() {
    this.openExternalLink();
  }

  openExternalLink() {
    window.open(this.url, '_blank');
    window.history.back();
  }

  render() {
    // language=HTML
    return html``;
  }
}

declare global {
  interface HTMLElementTagNameMap {
    'link-test-plugin': LinkTestPlugin;
  }
}
```

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [x] Minium required manager version: 23.09
- [x] Specific setting for review (eg., KB link, endpoint or how to setup)
- [x] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by yomybaby** [comment]
In sample plugin,` window.open('https://www.google.com', '_blank');` => `window.open(this.url, '_blank');` 😉

File: package.json [hunk 1]
```diff
@@ -147,6 +149,7 @@
     "redux": "^4.2.1",
     "redux-thunk": "^2.4.2",
     "rollup": "^3.29.3",
+    "rollup-plugin-multi-input": "^1.4.1",
```
Comment (yomybaby) [inline package.json]: Is this used? If not, please delete and run `npm i` to update `package-lock.json`. @agatha197

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (agatha197) [reply package.json]: Done!

**Review by yomybaby — APPROVED** [review]
LGTM!

**Pull request merged**